sections/main-collection-product-grid.liquid

```diff
@@ -82,7 +82,7 @@
       </aside>
     {%- endif -%}
 
-    <div class="product-grid-container{% if section.settings.filter_type == 'vertical' and section.settings.collapse_on_larger_devices %} product-grid-container-vertical{% endif %}" id="ProductGridContainer">
+    <div class="product-grid-container" id="ProductGridContainer">
```

Comment:
```suggestion
    <div class="product-grid-container{% if section.settings.filter_type == 'vertical' %} product-grid-container-vertical{% endif %}" id="ProductGridContainer">
```

Reply:
Yeah, we actually never had a `product-grid-container-vertical` class in the theme haha. 

This line was added in the same PR that removed the setting (#1443), so it's probably from some early explorations before the decision to remove the setting was made :D